Public/Src/Tools/Xldb.Proto/Events.proto

```diff
@@ -7,9 +7,9 @@ import "tools/google/protobuf/timestamp.proto";
 import "tools/google/protobuf/duration.proto";
 import "EnumsAndHelperStructs.proto";
 
-package BuildXL.Execution.Analyzer;
+package Xldb.Proto;
 
-option csharp_namespace = "BuildXL.Execution.Analyzer.Xldb";
+option csharp_namespace = "Xldb.Proto";

```

Comment:
>Xldb.Proto [](start = 27, length = 10)

I don't like that the namespace is just `Xldb.*`... I think it must be under the `BuildXL`, i.e., `BuildXL.Xldb`

Reply:
Should we rename the entire dll as well then to BuildXL.Xldb.dll? and similarly the nuget package?

---
In reply to: [316311200](https://github.com/microsoft/BuildXL/pull/775#discussion_r316311200) [](ancestors = 316311200)